### PR TITLE
option: ampExperimentToken

### PR DESCRIPTION
**Pull request by gilbox**
Adds a meta tag to the head to use AMP experiments. 

@ljharb @jeremyhuang3627

### CODE REVIEW

**Comment by jeremyhuang3627** [comment]
LGTM.

File: src/ampPage.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import ampScripts from './ampScripts';
 import ampJson from './ampJson';
 
+function ampExperimentToken(token) {
+  if (!token) return '';
+  return `<meta name="amp-experiment-token" content="${token}">\n`;
```
Comment (ljharb) [inline src/ampPage.js]: This has the same issue (which imo should have blocked the PR) which is that `token` isn't being sanitized, and could be used to accidentally or maliciously inject content into the page.

Comment (gilbox) [inline src/ampPage.js]: @ljharb if we are concerned about injection from values which should only be hardcoded by the programmer, then we need a PR to sanitize all of the options in this lib.

Comment (ljharb) [inline src/ampPage.js]: Yes, we do need that, ASAP.

**Pull request merged**